lib/models/package-info-cache/index.js

```diff
@@ -634,7 +635,7 @@ class PackageInfoCache {
       // directory from lower down in the project tree, and we want to get
       // the data into the cache ASAP. It may not necessarily be a 'real' error
       // if we find an issue, if nobody below is actually invoking the addon.
-      let nodeModules = this._readNodeModulesList(path.join(realPath, 'node_modules'));
+      let nodeModules = this._readNodeModulesList(`${realPath}/node_modules`);
```

Comment:
See my above comment - I think this *really* should use path.join().

Reply:
In internal code, I prefer to leave input as-is, until the normalization is actually required. This avoids wack-a-mole, as only code which cares always normalizes before utilization.